milestones.go

```diff
@@ -41,6 +41,7 @@ type Milestone struct {
 	StartDate   *ISOTime   `json:"start_date"`
 	DueDate     *ISOTime   `json:"due_date"`
 	State       string     `json:"state"`
+	WebURL      string     `json:"web_url"`
```

Comment:
I'm not seeing this field in the docs, but I'll trust you that it's in the actual response 👍

Reply:
Yeah, I was puzzled by this too, but seem to be a recurring field in a lot of resources now